vendor/github.com/openshift/api/config/v1/types_cluster_version.go

```diff
@@ -106,6 +111,33 @@ type ClusterVersionStatus struct {
 	AvailableUpdates []Update `json:"availableUpdates"`
 }
 
+type UpdateState string
+
+const (
+	CompletedUpdate UpdateState = "Completed"
+	PartialUpdate   UpdateState = "Partial"
+)
+
+type UpdateHistory struct {
+	// state reflects whether the update was fully applied
+	State UpdateState `json:"state"`
+
+	StartedTime    metav1.Time  `json:"startedTime"`
+	CompletionTime *metav1.Time `json:"completionTime"`
+
+	// version is a semantic versioning identifying the update version. When this
```

Comment:
can we just reuse `Update` or just embed it.. ?

Reply:
We could embed it, although sometimes I dislike embedding when the structs are used in different contexts to avoid accidentally getting a field we don't want.  I don't care too much.

Reply:
Best reason to have different is to have different godoc / apidoc.  I think in this case we want different godoc.